player.js

```diff
@@ -0,0 +1,58 @@
+Scrabble = require('./scrabble');
+var scrabble = new Scrabble();
+
+var Player = function(name){
+  this.plays = [];
+  this.name  = name;
+
+  this.play = function(word) {
+    if (this.hasWon === true) {
+      return false;
+    }
+    this.plays.push(word);
+    return this.plays;
+  };
+
+  this.total_score = function() {
+    var sum = 0;
+    for(var word of this.plays){
+      sum = sum + scrabble.score(word);
+    }
+    return sum;
+  };
+
+  this.hasWon = function() {
+    if (this.total_score() > 100 ){
+      return true;
+    }
+    return false;
+  };
+
+  this.highestScoringWord = function() {
+    var max = 0,
+        word_best = "";
+    for(var word of this.plays){
+      if (scrabble.score(word) > max) {
+        max = scrabble.score(word);
+        word_best = word;
+      }
+    }
+    return word_best;
+  };
+
+  this.highestWordScore = function() {
+    best_word = scrabble.highestScoreFrom(this.plays);
+    best_score = scrabble.score(best_word);
+    return best_score;
+  };
```

Comment:
you might consider renaming your functions to be more specific, since you do scoring in both the player and scrabble classes. something like ```playerHighestWordScore```. I also wonder if some of these scoring functions can be consolidated, especially if you return multiple things from one function (like ```{ word: score }```. it seems like there's some redundant calculations.